README.md

```diff
@@ -1,46 +1,35 @@
-# Sway Visual Studio Code Plugin
+<picture>
+   <source media="(prefers-color-scheme: light)" srcset="./images/light/sway.png">
+   <source media="(prefers-color-scheme: dark)" srcset="./images/dark/sway.png">
+   <img alt="Sway VSCode Plugin" src="./images/dark/sway.png">
+</picture>
 
 [![Visual Studio Marketplace Version](https://img.shields.io/visual-studio-marketplace/v/FuelLabs.sway-vscode-plugin)](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin)
 [![discord](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
 
-## When does the plugin get activated?
+This extension provides support for the Sway smart contract programming language.
 
-Currently it gets activated once you open a file with `.sw` extension.
+## Features
 
-## Testing
+- go to definition, implementation, type definition
+- symbol renaming
+- types and documentation on hover
+- inlay hints for types and parameter names
+- semantic syntax highlighting
 
-Prior to testing make sure you have [the Fuel toolchain](https://fuellabs.github.io/sway/latest/introduction/installation.html) installed as well.
+_Coming Soon_
 
-On macOS, ensure you have the `code` CLI tool installed by running `code --version`. If not, open the VSCode editor -> cmd + shift + p -> search `>Shell Command install` -> install. Additional information: <https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line>.
+- code completion with imports insertion
+- code actions
+- apply suggestions from errors
+- find all references, workspace symbol search
+- ... and many more
 
-## Testing as a real installed extension
+## Quick start
 
-```sh
-git clone git@github.com:FuelLabs/sway-vscode-plugin.git
-cd sway-vscode-plugin
-npm i
-npm run install-extension
-```
+1. Install the [Fuel toolchain](https://fuellabs.github.io/sway/latest/introduction/installation.html).
+1. Install the [Sway VSCode plugin](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin).
```

Comment:
2. Ensure `forc-lsp` is installed correctly by entering `forc-lsp --version`  into your terminal.
3. Install the ... (as above)